docs/tutorial.rst

```diff
@@ -1123,48 +1123,246 @@ We can then finally obtain the tree sequence::
+of the simulation, but rather the exact minimum that we need in order to complete
+the ancestry of the current generation.
+
+We run the coalescent simulation to complete this tree sequence using the
+``from_ts`` argument to :func:`.simulate`. Because we have simulated a
+two locus system with a recombination rate of ``1 / num_loci`` per generation
+in the Wright-Fisher model, we want to use the same system in the coalescent simulation.
+To do this we create recombination map using the
+:meth:`.RecombinationMap.uniform_map` class method to easily create a
+discrete map with the required number of loci.
+(Please see the :ref:`API <sec_api_simulate_from>` section for more details on the
+restrictions on recombination maps when completing an existing simulation.)
+We also use a ``Ne`` value of ``N / 2``
+since the Wright-Fisher simulation was haploid and ``msprime`` is diploid.
 
-..     site    node    derived_state
-..     0        14        1
+.. code-block:: python
 
-.. Here ``site`` is the index of the ``site`` at which the mutation occurred,
-.. ``node`` records the ID of the ancestral node associated with the mutation,
-.. and ``derived_state`` is the allele that any sample inheriting from that
-.. node at this site will have if another mutation does not intervene.  The
-.. ``node`` is not necessarily the ancestor in whom the mutation occurred, but
-.. rather the ancestor at the bottom of the branch in the tree at that site on
-.. which the mutation occurred.
+    recomb_map = msprime.RecombinationMap.uniform_map(num_loci, 1 / num_loci, num_loci)
+    coalesced_ts = msprime.simulate(
+        Ne=N / 2, from_ts=wf_ts, recombination_map=recomb_map, random_seed=5)
+
+
+
+After running this simulation we get the following trees::
+
+    interval =  (0.0, 1.0)
+                    26
+           ┏━━━━━━━━┻━━━━━━━┓
+           0                7
+           ┃                ┃
+           25               ┃
+      ┏━━━━┻━━━━┓           ┃
+      23        24          ┃
+    ┏━┻━┓    ┏━━╋━━━┓       ┃
+    ┃   21   ┃  ┃   22      20
+    ┃  ┏┻━┓  ┃  ┃  ┏┻━┓  ┏━━╋━━┓
+    10 14 19 11 18 15 17 12 13 16
+
+    interval =  (1.0, 2.0)
+                      28
+                 ┏━━━━┻━━━━━┓
+                 ┃          27
+                 ┃        ┏━┻━━┓
+                 26       ┃    ┃
+            ┏━━━━┻━━━━┓   ┃    ┃
+            0         7   4    8
+            ┃         ┃  ┏┻━┓  ┃
+            21        ┃  ┃  ┃  ┃
+    ┏━━┳━━┳━┻┳━━┳━━┓  ┃  ┃  ┃  ┃
+    14 19 10 13 16 18 12 15 17 11
+
+The trees have fully coalesced and we've successfully combined a forwards-time
+Wright-Fisher simulation with a coalescent simulation: hooray! However, these
+trees have some slightly odd properties which are important to be aware of.
+In both trees we can see that the old roots are still present in the trees,
+even through they have only one child and are clearly redundant. While this
```

Comment:
Insert here, maybe:
```
This is because the tables of ``from_ts`` have been retained, without modification, at the top of the tables of the output tree sequence. 
```